js/clients/lib/BcoinClient.js

```diff
@@ -93,7 +93,6 @@ class BcoinClient extends NodeClient {
       vout: txinfo.vout,
       locktime: txinfo.locktime,
       tx_id: txid,
-      tx_id_le: reverse(txid),
```

Comment:
these now return the BE format, right? we want that changed to LE

Reply:
Oh, yes, you're right